sample-plugin/pom.xml

```diff
@@ -26,28 +26,13 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
-            <!-- Require upper bound dependencies error between jenkins-test-harness and pipeline-model-definition -->
-            <dependency>
-                <groupId>net.sf.jopt-simple</groupId>
-                <artifactId>jopt-simple</artifactId>
-                <version>5.0.4</version>
-            </dependency>
```

Comment:
There is no longer a require upper bound dependencies error between `jenkins-test-harness` and `pipeline-model-definition`, so this workaround is no longer needed.